integration/error-boundary-v2-test.ts

```diff
@@ -409,7 +409,12 @@ test.describe("single fetch", () => {
         let app = new PlaywrightFixture(appFixture, page);
         await app.goto("/parent");
         await app.clickLink("/parent/child-with-boundary");
-        await waitForAndAssert(page, app, "#child-error", "CDN Error");
+        await waitForAndAssert(
+          page,
+          app,
+          "#child-error",
+          "Unable to decode turbo-stream response"
```

Comment:
See below, but we can't surface this underlying CDN error any longer with pre-rendering :/